src/main/java/com/JSONpostroute/services/SumDigitsService.java

```diff
@@ -0,0 +1,11 @@
+package com.JSONpostroute.services;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class SumDigitsService {
+    public static int sumDigits(int orderDigits) {
+        if(orderDigits ==0)return 0;
```

Comment:
Put this "== 0" instead of "==0".
Put "orderDigits % 10 + sumDigits(orderDigits / 10)" instead of "orderDigits%10 + sumDigits(orderDigits/10)"